src/UartSerial.cpp

```diff
@@ -122,7 +122,7 @@ UartSerial::begin (
   if ( B0 == baud_rate ) {
     ::perror("UartSerial::begin - Unsupported baud rate");
   // Validate configuration flags
-  } else if ( (size_t)-1 == c_cflags ) {
+  } else if ( (size_t)0 == c_cflags ) {
```

Comment:
This is a bad check. This would be the case for `SERIAL_5N1`. Can set a bit that is not considered by these flags and set in the `default:` case, then check for it's existence instead?